GOVERNANCE.md

```diff
@@ -0,0 +1,235 @@
+
+#### Code Owner Responsibilities
+Code Owners are responsible for reviewing and merging PRs for the projects and areas they are
+assigned to. They also
+- Participate in strategic planning, approve changes to the governance model, and manage the
+  copyrights within the project outputs.
+- Triage GitHub issues, applying [labels](https://github.com/llamastack/llama-stack/labels) to each new item. Labels are extremely useful for future issue follow ups. Adding labels is somewhat subjective, so please use your best judgment.
+- Make sure that ongoing PRs are moving forward at the right pace or closing them.
+
+
+### Becoming a Code Owner
+
+#### Who is eligible to become a Code Owner?
+Anyone can become a Code Owner. Typically, a potential Code Owner will need to show that they understand the project,
+its objectives, and its strategy. They will also have provided valuable contributions to the project over a period of
+time. Code Owners must also act in the interest of the community.
+
+#### Process
+Any existing Code Owner can nominate new Code Owners by opening a PR to the [CODEOWNERS](./.github/CODEOWNERS) file. Once they have been nominated,
+there will be a vote by the rest of the Code Owners. Code Owner voting is one of the few activities that takes place in
+private. This is to allow Code Owners to express their opinions about a nominee without causing embarrassment freely.
+The approval requires majority vote (>50%) from the Code Owners.
+
+#### Earning a Nomination
+
+The nomination is at the discretion of the existing Code Owners and there is no single path of earning a nomination for,
+however, we can give some guidance about some actions that would help:
+
+*   Start by expressing interest to the Code Owners that you are interested in becoming a Code Owner.
+*   You can start tackling issues labeled as ‘help wanted’, or if you are new to the project, some of the ‘good first issue’ tickets.
+*   As you gain experience with the codebase and our standards, we will ask you to do code reviews for incoming PRs (i.e., all Code Owners are expected to shoulder a proportional share of community reviews).
+*   We will expect you to start contributing increasingly complicated PRs, under the guidance of the existing Code Owners.
+
+#### Losing Code Owner Status
+
+If a Code Owner is no longer interested and cannot perform the Code Owner duties listed above, they can volunteer to be
+moved to emeritus status. The Code Owner status is attributed for life otherwise. An emeritus Code Owner may request
+reinstatement of commit access from the rest of Code Owners. Such reinstatement is subject to lazy consensus approval of
+active Code Owners.
+
+Emeritus status is a nominal title, and confers no special rights (like voting) or access. Emeritus members are
+functionally identical to normal contributors, with the exception that they can request for reinstatement of their
+commit access.
```

Comment:
maybe just say "codeowner access" - since access would be to discord channels, gitops workflows, in addition to commit access.